packages/main/src/components/ObjectPage/ObjectPageAnchorButton.tsx

```diff
@@ -19,40 +19,38 @@ interface ObjectPageAnchorPropTypes {
   mode: ObjectPageMode;
 }
 
-const useStyles = createUseStyles<string>(
-  ({ parameters }: JSSTheme) => ({
-    anchorButtonContainer: {
-      position: 'relative',
-      display: 'inline-flex',
-      alignItems: 'center',
-      cursor: 'pointer',
-      '&:not(:first-child)': {
-        marginLeft: '2rem'
-      }
-    },
-    button: {
-      color: parameters.sapUiContentLabelColor,
-      fontFamily: fonts.sapUiFontFamily,
-      fontSize: fonts.sapMFontMediumSize
-    },
-    selected: {
-      color: parameters.sapUiSelected,
-      minWidth: '2rem',
-      textAlign: 'center',
-      '&:after': {
-        content: '""',
-        borderBottom: `0.188rem solid ${parameters.sapUiSelected}`,
-        width: '100%',
-        position: 'absolute',
-        bottom: 0,
-        left: 0
-      }
+const anchorButtonStyles = ({ parameters }: JSSTheme) => ({
+  anchorButtonContainer: {
+    position: 'relative',
+    display: 'inline-flex',
+    alignItems: 'center',
+    cursor: 'pointer',
+    '&:not(:first-child)': {
+      marginLeft: '2rem'
+    }
+  },
+  button: {
+    color: parameters.sapUiContentLabelColor,
+    fontFamily: fonts.sapUiFontFamily,
+    fontSize: fonts.sapMFontMediumSize
+  },
+  selected: {
+    color: parameters.sapUiSelected,
+    minWidth: '2rem',
+    textAlign: 'center',
+    '&:after': {
+      content: '""',
+      borderBottom: `0.188rem solid ${parameters.sapUiSelected}`,
+      width: '100%',
+      position: 'absolute',
+      bottom: 0,
+      left: 0
     }
-  }),
-  {
-    name: 'ObjectPageAnchorButton'
   }
-);
+});
+const useStyles = createUseStyles<JSSTheme, keyof ReturnType<typeof anchorButtonStyles>>(anchorButtonStyles, {
```

Comment:
Is there a way to somehow abstract that typing mess? It looks pretty tedious to write in every component :/

Reply:
I tried but it did not work...
`export type JssReturnType<T> = keyof ReturnType<typeof T>` but this is not valid because is a type and a value at the same time...